components/UI/Map/utils.ts

```diff
@@ -18,3 +18,19 @@ export const localStorageKeys = {
 export const localForageKeys = {
   markersVisited: "markersVisited",
 } as const;
+
+export const safeGetLocalStorage = (key: string) => {
```

Comment:
I think these functions are not map utils. Putting them under `utils` folder makes more sense. If you're agree with me I can do this change tonigh.